database/migrations/2024_04_19_111943_add_collaborations_to_resumes_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Query\Expression;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('resumes', function (Blueprint $table) {
+            $table->json('collaborations_ids')->default(new Expression('(JSON_ARRAY())'));
```

Comment:
I think this field should be nullable:

``$table->json('collaborations_ids')->default(new Expression('(JSON_ARRAY())'))->nullable();``